### ISSUE TITLE
feature request: tiling

### ISSUE THREAD

**Issue by hasufell** [dm0-/window_merge#2]
Hey, nice plugin

how about supporting tiling like I make the buddylist fullscreen with this nice plugin and can have 2 or 4 windows at once instead of tabs.

+1 user

**Comment by BWMerlin** [comment]
I like the sound of it if we could group things and have them stick to certain title groups ie. have one tile all irc chats, another all IM's another what ever else and so on.

**Comment by dm0-** [comment]
I agree that tiling would be a nice feature, especially for more efficient use of widescreen displays, etc.  Looking around the Pidgin source a bit more (without yet actually writing any code), I believe this functionality will be feasible to implement.  However, I think conversation tiling is slightly outside the scope of this project and warrants its own plugin.

I wanted to have something working before replying to this, but I've been fairly busy lately.  Hopefully within a week or two I can have something usable uploaded.  (I will obviously ensure both plugins can be enabled simultaneously.)

**Comment by dm0-** [comment]
I feel another note on this is due since the date I mentioned is long past.  I am still alive and occasionally working on the tiling plugin among other things in my down time.  Something functional is created (including changes for this plugin to be compatible with it) but it's still too unstable to start distributing.  I will update again when I think the plugin is in a state where an average user would be unlikely to break it.

**Comment by BWMerlin** [comment]
Thanks for the update, look forward to when it is finished.

**Comment by hasufell** [comment]
+1 :)
